### ISSUE TITLE
Takes a long time for verifyelementpresent and assertelementpresent and eventually failed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#13]
```
Operating system :Windows XP SP3
.Net Framework version :3.5 SP1
Office Version :2010
SeleniumWrapper version :1.0.6.3

What steps will reproduce the problem?

Public selenium As New SeleniumWrapper.WebDriver
Sub Test()
selenium.Start "firefox", "http://bing.com"
selenium.Open "/"
MsgBox selenium.verifyElementPresent("//tr/td[@value='google']")
End Sub

What is the expected output? - True or False.
What do you see instead? - timeout.

Please provide any additional information below:
It happens on verifyelementpresent, assertelementpresent, verifytextpresent, 
asserttextpresent, verifytext, asserttext...

```

Original issue reported on code.google.com by `zope...@gmail.com` on 21 Dec 2012 at 6:06

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in version 1.0.8.0
Could you tell me if you still get the error?
```

Original comment by `florentbr` on 6 Jan 2013 at 12:08
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
It's fixed in version 1.0.8.0
Thanks for your excelent work!
```

Original comment by `zope...@gmail.com` on 21 Jan 2013 at 9:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `florentbr` on 21 Jan 2013 at 1:45
- Changed state: **Fixed**
